packages/mobile/src/components/skeleton/Skeleton.tsx

```diff
@@ -87,4 +87,6 @@ export const StaticSkeleton = (props: StaticSkeletonProps) => {
   return <Skeleton noShimmer {...props} />
 }
 
+export { Skeleton }
```

Comment:
Were there too many references to skeleton to replace the default export?

Reply:
yeah was just a stop gap for now, but yeah should go in and remove default exports eventually